.changelog

```diff
@@ -0,0 +1,4 @@
+# - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
```

Comment:
File was created because of misleading (outdated) statements in the README file.  Can you please delete it again, if not already done?